tests/compilers/test_ast.py

```diff
@@ -211,12 +211,25 @@ def test_ast_bad_assert():
 def test_ast_good_global():
     "Make sure AST can compile valid global"
     can_compile("(global a)")
+    can_compile("(global foo bar)")
 
 
 def test_ast_bad_global():
     "Make sure AST can't compile invalid global"
     cant_compile("(global)")
-    cant_compile("(global foo bar)")
+    cant_compile("(global (foo))")
+
+
+if PY3:
```

Comment:
so skipping won't work great if we raise a SyntaxError -- which is why we have the if-python2 macro